WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/GroupedProductListViewModel.kt

```diff
@@ -119,15 +142,22 @@ class GroupedProductListViewModel @AssistedInject constructor(
         )
     }
 
-    private fun getGroupedProductIds() = navArgs.groupedProductIds.split(",").map { it.toLong() }
+    private fun getOriginalGroupedProductIds(): List<Long> {
+        return if (navArgs.groupedProductIds.isNotEmpty()) {
+            navArgs.groupedProductIds.split(",").map { it.toLong() }
+        } else emptyList()
+    }
```

Comment:
np: since this is a getter expression and looks like an one time value transformation, you can turn this expression into a `val` property and took advantage of that to add a layer of protection for the split and map operations, but also keeping the transformed value instead of regenerating it for every method call, if you think it makes sense, something like:

```suggestion
        private val originalGroupedProductIds = 
        navArgs.groupedProductIds
            .takeIf { it.isNotEmpty() }
            ?.split(",")
            ?.mapNotNull { it.toLongOrNull() }
            .orEmpty()
```

This kind of refactoring will probably require to move the `val` declaration to the top of the class to avoid initialization issues, in case you decide to go with this suggestion